api/types/swarm/task.go

```diff
@@ -40,13 +40,14 @@ type Task struct {
 	Meta
 	Annotations
 
-	Spec                TaskSpec            `json:",omitempty"`
-	ServiceID           string              `json:",omitempty"`
-	Slot                int                 `json:",omitempty"`
-	NodeID              string              `json:",omitempty"`
-	Status              TaskStatus          `json:",omitempty"`
-	DesiredState        TaskState           `json:",omitempty"`
-	NetworksAttachments []NetworkAttachment `json:",omitempty"`
+	Spec                     TaskSpec            `json:",omitempty"`
+	ServiceID                string              `json:",omitempty"`
+	Slot                     int                 `json:",omitempty"`
+	NodeID                   string              `json:",omitempty"`
+	Status                   TaskStatus          `json:",omitempty"`
+	DesiredState             TaskState           `json:",omitempty"`
+	NetworksAttachments      []NetworkAttachment `json:",omitempty"`
+	AssignedGenericResources []*GenericResource  `json:",omitempty"`
```

Comment:
`Resources` might even be okay. We'd have to flow some of this back to swarmkit (these names are a little clearer).

@aaronlehmann Thoughts? I know this has already been through a large review cycle.